menu.rb

```diff
@@ -0,0 +1,148 @@
+# Brandi Wilson - RANDOM MENU GENERATOR (Dessert)
+
+# welcome & prompt
+question = "Would you like to see our menu (1) or create your own (2)?"
+puts "WELCOME TO FOOD WORLD".center(question.length)
+
+# list of words for the random menu
+food_adj = ["Gooey", "Crumbley", "Fun", "Questionable", "Mysterious", "Rainbow", "Mutant", "Magical", "Chilly", "Cheesy", "Spikey", "Cute", "Smooth", "Happy", "Hilarious", "Cat-loved", "Radioactive", "Watery", "Numeric", "Confused"]
+food_loc = ["Chinese", "Japanese", "British", "Scottish", "French", "Korean", "Indian", "Hawaiian", "American", "Canadian", "German", "Portugese", "Vietnamese", "Irish", "Italian", "African", "Australian", "Lihuanian", "Brazilian", "Mexican", "Egyptian"]
+food = ["Chicken", "Duck", "Baked Potato", "Eggplant", "Pie", "Ice Cream", "Fish", "Bread", "Mold", "Lettice", "Donut", "Coffee", "Tea", "Water", "Smoothie", "Burger", "Pizza", "Corn", "Rice", "Spaghetti", "Salad", "Apple"]
+
+# sorts the above arrays by length & assigns the shortest array length as the max number of available menu items
+max_lengths = [food_adj.length, food_loc.length, food.length].sort
```

Comment:
I think you could collapse these two lines into one by using the `min` method rather than using sort and then pulling the first index

http://ruby-doc.org/core-2.2.2/Enumerable.html#method-i-min